root/application/src/main/java/at/fhv/itb5c/rmi/application/dto/UserConverter.java

```diff
@@ -0,0 +1,40 @@
+package at.fhv.itb5c.rmi.application.dto;
+
+import at.fhv.itb5c.commons.dto.IUser;
+import at.fhv.itb5c.model.entity.User;
+
+public class UserConverter {
+	public static IUser toDTO(User user){
+		UserDTO userdto = new UserDTO();
+		userdto.setId(user.getId());
```

Comment:
Concerning the topic of checking if every single parameter of every method is null: Does that really make sense? Maybe we could just use [Objects.requireNonNull](http://docs.oracle.com/javase/7/docs/api/java/util/Objects.html#requireNonNull(T)) to check at the beginning of a method if all parameters are set, and call it a day?